modules/nw-exposing-router-metrics.adoc

```diff
@@ -0,0 +1,157 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress_operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-exposing-router-metrics_{context}"]
+= Exposing router metrics
+
+You can expose the HAProxy router metrics by default in Prometheus format on the default stats port, 1936. The external metrics collection and aggregation systems such as Prometheus and statsd can access the HAProxy router metrics. You can view the HAProxy router metrics in a browser in the HTML and cluster service version (CSV) format.
+
+.Prerequisites
+
+* You configured your firewall to access the default stats port, 1936.
+
+.Procedure
+
+. A router pod creates a default username and password for the router. A router pod stores username and password at `/var/lib/haproxy/conf/metrics-auth/statsUsername` and `/var/lib/haproxy/conf/metrics-auth/statsPassword` respectively.
+
+.. Get the username by running the following command:
++
+[source,terminal]
+----
+$ oc rsh <router_pod> cat metrics-auth/statsUsername
+----
++
+.Example command
+[source,terminal]
+----
+$ oc rsh router-default-d78cffbc6-jcgfn cat metrics-auth/statsUsername
+----
++
+.Example output
+[source,terminal]
+----
+dXNlcmN3NnY2
+----
+
+.. Get the password by running the following command:
++
+[source,terminal]
+----
+$ oc rsh <router_pod> cat metrics-auth/statsPassword
+----
++
+.Example command
+[source,terminal]
+----
+$ oc rsh router-default-d78cffbc6-jcgfn cat metrics-auth/statsPassword
+----
++
+.Example output
+[source,terminal]
+----
+cGFzczdqbnJt
+----
+
+. Get the raw statistics in Prometheus format by running the following command:
++
+[source,terminal]
+----
+$ curl -u <user>:<password> http://<router_IP>:<stats_port>/metrics
```

Comment:
Think the `IP` should still be lowercase `<router_ip>` (here and the remaining that follow) per user-replaceable values guidelines.